versioned_docs/version-0.9.0/tests-integration/unit-tests.mdx

```diff
@@ -0,0 +1,166 @@
+---
+id: unit-tests
+title: Unit Tests
+---
+
+# Unit Tests
+
+import versions from "@site/static/files/0.9.0/versions.json"
+import { Codeblock } from "@components"
+
+This page walks you through Drill4J integration for Java-based unit tests.
+It will enable you to use such features as _Per Test Coverage_ and _Test Recommendations_.
+
+## Prerequisites
+
+1. This page assumes you have run core Drill4J Services. For instructions see:
+
+    - [Getting Started](/docs/getting-started/local-deployment)
+
+2. Unit tests launched with either Maven or Gradle
+
+This instruction involves making changes to the Gradle/Maven project build file.
+
+## Install Drill4J Plugin
+
+The Drill4J CI/CD integration plugin allows you to run unit tests together with the Test Agent and Application Agent.
```

Comment:
I suggest to rephrase this sentence.
"Drill4J CI/CD Integration Plugin allows to enable both Test Agent and App Agent to capture metrics from Unit tests execution"